fe/fe-core/src/main/java/com/starrocks/system/SystemInfoService.java

```diff
@@ -728,23 +737,26 @@ public Stream<ComputeNode> backendAndComputeNodeStream() {
 
     public List<Long> seqChooseBackendIdsByStorageMedium(int backendNum, boolean needAvailable, boolean isCreate,
                                                          TStorageMedium storageMedium) {
-        final List<Backend> backends =
+        final List<ComputeNode> backends =
```

Comment:
I don't think we should change any backend to compute node in SystemInfo module. It is safe to assume that backend is a compute node, so when retrieving compute node, responds back with backend node, would be still OK. But it is not OK when the caller wants a BE, SystemInfo just respond back Compute nodes back as BE.